test/hitchhiker/tree/messaging_test.clj

```diff
@@ -153,9 +153,9 @@
   (mixed-op-seq 0.5 250 5000))
 
 (defspec test-many-keys-bigger-trees
-  1000
```

Comment:
Actually, I'm mistaken. The newer clojure `test.generative` supports this, but this `test.check` does not. Right before the final commit, could you change this back to 1000? Thank you!